### PR TITLE
Add new page for JIT explanation

### PR DESCRIPTION
**Pull request by bwillis**
This is a new page for explaining the various options for JIT provisioning-long overdue.

### CODE REVIEW

**Comment by bwillis** [comment]
@stacyspiva question: when developing locally, I noticed that most of the images don't load, is that expected? Is there another way to verify it's working?

**Comment by jobertabma** [comment]
@bwillis Changes looks fine to me. Read through the docs and they're clear to me. I'd suggest @stacyspiva to read them too before we merge!

**Comment by stacyspiva** [comment]
@bwillis the images should load. Were they added to the `images` folder?

**Comment by bwillis** [comment]
> @bwillis the images should load. Were they added to the `images` folder?

Hmm yes they were. I didn't see any loading for me:

Screen Shot 2020-05-28 at 10 42 24 AM

File: docs/programs/sso-jit.md [hunk 1]
```diff
@@ -8,41 +8,22 @@ HackerOne offers Just-in-time (JIT) provisioning with [SSO via SAML](single-sign
...
 
-All SAML users have access to the platform by default, but do not necessarily have access to programs. This will help you understand the options that are available to you when configuring your SAML settings.
+HackerOne can confirm the mapping between the SSO provider and HackerOne is done correctly by inspecting the assertion statement in the SAML Response:
```
Comment (bwillis) [inline docs/programs/sso-jit.md:25]: This part applies to advanced setup only, but appears to have fallen out of the table:
Screen Shot 2020-05-28 at 1 57 23 PM

Comment (stacyspiva) [reply docs/programs/sso-jit.md:25]: I'd like to add it to the table, but I don't think there's a clean way to add a code block to a table in markdown. Is it ok if we just leave it out of the table?

Comment (bwillis) [reply docs/programs/sso-jit.md]: Ok, I think it still works outside of the table, made a small copy adjustment. Thanks!

**Pull request merged**